lib/bcdice/game_system/ConvictorDrive.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+require 'bcdice/base'
+
+module BCDice
+  module GameSystem
+    class ConvictorDrive < Base
+      # ゲームシステムの識別子
+      ID = 'ConvictorDrive'
+
+      # ゲームシステム名
+      NAME = 'コンヴィクター・ドライブ'
+
+      # ゲームシステム名の読みがな
+      #
+      # 「ゲームシステム名の読みがなの設定方法」（docs/dicebot_sort_key.md）を参考にして
+      # 設定してください
+      SORT_KEY = 'こんういくたあとらいふ'
+
+      # ダイスボットの使い方
+      HELP_MESSAGE = <<~MESSAGETEXT
+        xCD@z>=y: x個の10面ダイスで目標値y、クリティカルラインzの判定を行う
+        SLT: 技能レベル表を振る
+        DCT: 遅延イベント表を振る
+      MESSAGETEXT
+
+      # ダイスボットで使用するコマンドを配列で列挙する
+      register_prefix("[-+*0-9\(\)]*CD", "SLT", "DCT")
+
+      def initialize(command)
+        super(command)
+
+        @sides_implicit_d = 10
+      end
+
+      def eval_game_system_specific_command(command)
+        debug("eval_game_system_specific_command Begin")
+
+        case command
+        when "SLT"
+          return skill_rank_table
+        when "DCT"
+          return delay_event_table
+        else
+          return check_command(command)
+        end
+      end
+
+      def check_command(command)
+        parser = Command::Parser.new('CD', round_type: round_type)
+                                .has_prefix_number
+                                .enable_critical
+                                .restrict_cmp_op_to(:>=)
+        cmd = parser.parse(command)
+
+        unless cmd
+          return nil
+        end
+
+        dice_list = @randomizer.roll_barabara(cmd.prefix_number, 10)
+        critical = (cmd.critical || 10).clamp(cmd.target_number, 10)
```

Comment:
```suggestion
        critical = cmd.critical&.clamp(cmd.target_number, 10) || 10
```
`&.` (safe navigation operator) を使うと綺麗に書けます